library/src/main/java/com/davemorrissey/labs/subscaleview/SubsamplingScaleImageView.java

```diff
@@ -685,12 +685,15 @@ public boolean onTouchEvent(@NonNull MotionEvent event) {
         }
 
         if (vTranslateStart == null) { vTranslateStart = new PointF(0, 0); }
-        if (vTranslateBefore == null) { vTranslateBefore = new PointF(0, 0); }
+        if (vTranslateBefore == null) {
+            vTranslateBefore = new PointF(vTranslate.x, vTranslate.y);
+        } else {
```

Comment:
I added this part to maintain the exact same behavior as before but I think just assigning a new `PointF` instance will be enough if you're not worried about the overhead and GC cost